tests/wpt/metadata/html/dom/interfaces.https.html.ini

```diff
@@ -1017,7 +1017,7 @@
     expected: FAIL
 
   [HTMLMediaElement interface: document.createElement("video") must inherit property "crossOrigin" with the proper type]
-    expected: FAIL
+    expected: PASS
```

Comment:
For these PASSes, instead of removing the file, you can just remove the line above and this line:
```suggestion
```